packages/editor/src/editor/properties/PropertyItem.tsx

```diff
@@ -17,6 +20,21 @@ type PropertyItemProps = {
 const toString = (primitive?: PrimitiveValue) => `${primitive ?? ''}`;
 const toNumber = (primitive?: PrimitiveValue) => Number(primitive ?? 0);
 const toBoolean = (primitive?: PrimitiveValue) => Boolean(primitive ?? false);
+const toArray = (primitive?: PrimitiveValue): unknown[] => (Array.isArray(primitive) ? primitive : []);
+
+const selectTablecolumns: ColumnDef<SelectItem, string>[] = [
+  {
+    accessorKey: 'label',
+    header: () => <span>Label</span>,
+    cell: cell => <InputCellWithBrowser cell={cell} />,
+    minSize: 50
+  },
+  {
+    accessorKey: 'value',
+    header: () => <span>Value</span>,
+    cell: cell => <InputCellWithBrowser cell={cell} />
+  }
+];
 
```

Comment:
why here and not in the TableField component?

Reply:
or better I understand why you don't want them in the TableField component to hold this generic. But then add a new SelectTableField which is a wrapper around the TableField with this column def.

Reply:
Yes exactly, the reason why I have them outside the TableField component is so that I can reuse TableField better (e.g. Radio-Component/Combox-Component). Thanks good input, I will make a wrapper component 👍